external/upstream/libxc/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ else()
         # Default: use a stable release tarball of libxc. To use the
         # development version of libxc, instead, comment the URL line,
         # and uncomment the GIT lines.
-        URL https://gitlab.com/libxc/libxc/-/archive/6.2.2/libxc-6.2.2.tar.gz
+        URL https://gitlab.com/libxc/libxc/-/archive/7.0.0/libxc-7.0.0.tar.gz
```

Comment:
so either v6 or v7 should work at build-time? 

iirc, libxc has SameMajorVersion for its cmake tolerant version range. since `find_package(libxc 6.2)` it'll likely reject 7, so we'll need to put in the extra logic psi-side to allow 6 or 7 like we had earlier in the year for allow 5 or 6. also, the codedeps.yaml at top level will likely need edits. these are mostly notes to myself and/or explanation if CI fails.

Reply:
Yeah the API hasn't changed, only the ABI.